Documentation/Configuration/ConfigurationFiles.rst

```diff
@@ -0,0 +1,26 @@
+.. include:: /Includes.rst.txt
+
+.. _configuration-files:
+
+===================
+Configuration files
```

Comment:
I suggest to do this change in a separate PR, so it can be merged also to 11.5. This has nothing to do with cms-composer-installers v5.

Reply:
I only moved the information which was in hidden in  the directory structure before. So that I can link from both  Compsoser and Legacy installatins to it

Reply:
I will make a follow up for v11 then